File: src/targets/javascript/axios/fixtures/multipart-file.js
import axios from 'axios';

const form = new FormData();
form.append('foo', 'src/fixtures/files/hello.txt');

const options = {
  method: 'POST',
  url: 'https://httpbin.org/anything',
  headers: {'content-type': 'multipart/form-data; boundary=---011000010111000001101001'},
  data: '[form]'
};

try {
  const { data } = await axios.request(options);
  console.log(data);
} catch (error) {
  console.error(error);
}

File: src/targets/java/asynchttp/fixtures/multipart-file.java
AsyncHttpClient client = new DefaultAsyncHttpClient();
client.prepare("POST", "https://httpbin.org/anything")
  .setHeader("content-type", "multipart/form-data; boundary=---011000010111000001101001")
  .setBody("-----011000010111000001101001\r\nContent-Disposition: form-data; name=\"foo\"; filename=\"src/fixtures/files/hello.txt\"\r\nContent-Type: text/plain\r\n\r\n\r\n-----011000010111000001101001--")
  .execute()
  .toCompletableFuture()
  .thenAccept(System.out::println)
  .join();

client.close();